temporal/api/cloud/nexus/v1/message.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package temporal.api.cloud.nexus.v1;
+
+option go_package = "go.temporal.io/api/cloud/nexus/v1;nexus";
+
+import "google/protobuf/timestamp.proto";
+
+message EndpointSpec {
+    // The name of the endpoint. Must be unique within an account.
+    // The name must match `[a-zA-Z_][a-zA-Z0-9_]*`.
+    // This field is mutable.
+    string name = 1;
+
+    // Indicates where the endpoint should forward received nexus requests to. 
+    EndpointTargetSpec endpoint_target_spec = 2;
+
+    // The set of policies (e.g. authorization) for the endpoint. Each request's caller
+    // must match with at least one of the specs to be accepted by the endpoint.
+    // This field is mutable.
+    repeated EndpointPolicySpec policy_specs = 3;
+
+    // Optional description. Can be in markdown for rendering in the UI.
+    // Note that this is an experimental field that may be deprecated in the future
+    string description = 4;
```

Comment:
We support markdown-based descriptions in API elements? We have some other work trying to define the subset of markdown we're going to support. If we don't have description on namespaces or users or whatever, can we wait until we can consistently provide description on all of these types of resources instead of making Nexus endpoint the first? I support the idea, just think it should be generally applicable.

Reply:
this is definitely open/experimental. we dont support description on any other resources (except for service accounts, which is also `string description`, and thats not released yet). Am fine removing this for now and adding it later. It's also why it was marked as "experimental" via a comment. @bergundy - any thoughts here?

Reply:
removed for now. can add it in a separate PR after more discussion

Reply:
I missed this conversation.

Our designs all include description in markdown, we have to keep this field.
I have requested that we make it a `Payload` so eventually, when we support account level codecs in the UI we can encrypt them.